x-pack/plugins/case/kibana.json

```diff
@@ -2,7 +2,7 @@
   "configPath": ["xpack", "case"],
   "id": "case",
   "kibanaVersion": "kibana",
-  "requiredPlugins": ["security", "actions"],
+  "requiredPlugins": [],
```

Comment:
Are they enabled by default?

Reply:
because we are not using actions in the API anymore and also we want to make sure people can use case without security

Reply:
@cnasikas Good catch on this one, you were right. We need to keep actions as a required plugin since we are using it to get all the connectors for `.servicenow`